wire-ios-sync-engine/Tests/Source/Integration/ConnectionTests.m

```diff
@@ -612,7 +612,7 @@ - (void)DISABLED_testThatWeDontSeeASystemMessageWhenAUserAcceptsAConnectionReque
         XCTAssertEqual(conv1.conversationType, ZMConversationTypeOneOnOne);
         XCTAssertEqual(conv1.allMessages.count, 1u, @"%@", [conv1 lastMessagesWithLimit:50]);
         ZMSystemMessage *message1 = (ZMSystemMessage *)conv1.lastMessage;
-        XCTAssertEqual(message1.systemMessageType, ZMSystemMessageTypeConnectionRequest);
+       // XCTAssertEqual(message1.systemMessageType, ZMSystemMessageTypeConnectionRequest);
```

Comment:
So the test could run. Someone removed or deprecated the system message but it was still referenced here. I'm not sure if we need that test since that systemMessage has been removed.